cmd/mirror_list.go

```diff
@@ -31,7 +43,9 @@ func aptlyMirrorList(cmd *commander.Command, args []string) error {
 
 	context.CloseDatabase()
 
-	sort.Strings(repos)
+	if len(repos) > 0 {
```

Comment:
`sort.Strings` works on empty lists, too. There is no need for this "if" statement.